package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
```

Comment:
Just curious what this change does?

Reply:
> Just curious what this change does?

I deleted the up arrow to see if there was any difference in functionality. It appears to still run the same but the change was caused when I ran npm install react-scripts.